src/components/RemoveButton.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+
+const RemoveButton = ({ bookId }) => {
+  const dispatch = useDispatch();
+
+  const handleRemoveBook = () => {
+    dispatch(removeBook(bookId));
+  };
+
+  return (
+    <button type="button" onClick={handleRemoveBook}>
+      Remove <span className="vertical-line">|</span>
+    </button>
+  );
+};
+
+RemoveButton.propTypes = {
+  bookId: PropTypes.number.isRequired,
+};
+
+export default RemoveButton;
```

Comment:
- [ ] Looking at your `.eslintrc.json`, it seems you are using the wrong or modified code, The content of the file is different from the one in the official Microverse Linters repository and it affects the Linters check, some errors in the code are not picked up by linters. I'll suggest you go to this repository [ https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json ] and copy the correct `.eslintrc.json` file content. Make sure to run the linters check after that and fix any error that your code has. NB: It is against Microverse rules to Modify any Linters code, the linters are there to make sure your code follows the best standard. :smile: :wave: :+1: